test/python/common.py

```diff
@@ -5,16 +5,20 @@
 # This source code is licensed under the Apache License, Version 2.0 found in
 # the LICENSE.txt file in the root directory of this source tree.
 
+# pylint: disable=invalid-name
```

Comment:
Can you move this to inside the `assertStatus` method, to avoid the file-level disable obscure other potential naming warnings in the file? The test files are usually a bit of a wild territory when it comes to linting and usually we are not that picky with the disables, but in this particular file, there is value in keeping it as neat as possible.

Eventually we should tweak the `.pylintrc` so the `assertFoo` methods do not raise this error, as this is one of the main reasons the disable is in place in the test files - food for another PR.